### DISCUSSION TITLE
C and CXX compiler identification unknown

### DISCUSSION THREAD

**Discussion by vishal203202** [trustwallet/wallet-core#3525] [Q&A]
**Describe the bug**
When attempting to build wallet-core using the provided [Docker image installation](https://developer.trustwallet.com/developer/wallet-core/developing-the-library/building#building-inside-docker-image) , the build process encounters an issue related to the C and C++ compilers. 

**To Reproduce**

1. Use the provided [Docker image installation](https://developer.trustwallet.com/developer/wallet-core/developing-the-library/building#building-inside-docker-image) from the wallet-core website.
2. Attempt to build Trust Wallet using the provided instructions.
3. Observe the error mentioned above during the build process.

**Expected behavior**
The build process should be successful and not encounter issues related to the C and C++ compilers.

**Screenshots**
![Screenshot from 2023-10-30 23-45-28](https://github.com/trustwallet/wallet-core/assets/101107510/7c98e318-3821-4d84-8514-b5d08a89ea6b)

**Additional context**
It appears that the CMake configuration is unable to detect the C and C++ compilers properly. The error message suggests that the clang and clang++ compilers are not found in the PATH. This issue prevents the successful build of Trust Wallet using the provided Docker image.

**Comment by satoshiotomakan** [comment]
Could you please provide which version of wallet-core you are trying to build, the exact commands?
Did you make any changes in Dockerfile?

**Comment by vishal203202** [comment]
First, I cloned the latest commit of wallet-core repository and tried to build the project locally on docker following the instructions given on https://developer.trustwallet.com/developer/wallet-core/developing-the-library/building#building-inside-docker-image. I haven't made any changes to the Dockerfile.
